.circleci/Makefile

```diff
@@ -0,0 +1,21 @@
+#PROJECT ?= istio-testing
+VERSION ?= 0.4.4
+#HUB ?= gcr.io/
+
+PROJECT ?= costinm
```

Comment:
That can't be the right default

Reply:
I've used this for testing - not sure if we should use gcr.io or dockerhub (istio/istio is using the later), 
and what name to use ( 'ci' is used by istio/istio)